decidim-core/app/cells/decidim/amendable/announcement_cell.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Decidim::Amendable
+  # This cell renders the callout with information about the state of the emendation
+  class AnnouncementCell < Decidim::ViewModel
+
+    def show
+      cell "decidim/announcement", announcement
+    end
+
+    private
+
+    def announcement
+      {
+      announcement: emendation_message,
+      callout_class: state_classes
+      }
+    end
+
+    def emendation_message
+      case model.emendation_state
```

Comment:
I think you can just pass many params to `t` and if it's not used is OK. This way you don't need the `case` statement.

Reply:
:+1: